### ISSUE TITLE
./pythonscript/bin/pip complains about python3 not being an executable

### ISSUE THREAD

**Issue by touilleMan** [touilleMan/godot-python#155]
tl;dr: use `./pythonscript/bin/python3 -m pip` instead

see #153

```shell
$ cd examples/pong
$ ./pythonscript/bin/pip
Failed to execute process './pythonscript/bin/pip'. Reason:
The file './pythonscript/bin/pip' specified the interpreter '/build/out/python/install/bin/python3', which is not an executable command.
```

**Comment by touilleMan** [comment]
Situation as of version 0.40.0 is all scripts commands (pip, easy_install, 2to3 etc.) are still broken

On linux&macOS those commands have an invalid shabang (e.g. `#!/install/bin/python3.8` on linux) that refers to the prefix configuration provided during Python interpreter build.
A simple solution would be to do a sed in the scripts to change this path, but shabang path must be absolute (and a Godot project using godot-python can be located anywhere !).
It seems [there is some hacky ways to achieve relative shabang](https://stackoverflow.com/questions/20095351/shebang-use-interpreter-relative-to-the-script-path) so we may try this in the future.

On Windows the scripts come as .exe files so editing them is even harder :(

For the moment I've left those broken scripts as part of the release. The reason is there are simple to fix on linux&macOS for the final user (just edit the shabang), and they might be needed by 3rd party (especially `python3.8-config` this is often use to retrieve the compile/link flags needed to build native modules)

It would be greate to add a README in the folder though to explain this mess ;-)

**Comment by whogben** [comment]
@touilleMan I'd like to update the shebang to be valid on mac to use pip, can you dumb it down for me a bit more on what the simple fix is?

- what file to edit the shebang in?
- what file to put the absolute path to in the shebang?

When I try executing the file (sudo ./pythonscript/osx-64/bin/python3 -m pip) it replies "command not found"
I looked inside pythonscript/osx-64/bin/python3 but there's no shebang inside, just "python3.8" in the file.

PS: many thanks for this incredible project, really enabling some super cool stuff.
